castai/data_source_castai_cluster.go

```diff
@@ -111,9 +216,79 @@ func dataSourceCastaiClusterRead(ctx context.Context, data *schema.ResourceData,
 		data.Set(ClusterFieldKubeconfig, []interface{}{})
 	}
 
+	policies, err := client.GetPoliciesWithResponse(ctx, sdk.ClusterId(data.Id()))
+	if checkErr := sdk.CheckGetResponse(policies, err); checkErr == nil {
+		log.Printf("[INFO] Autoscaling policies for cluster %q", data.Id())
+		data.Set(PolicyFieldAutoscalerPolicies, flattenAutoscalerPolicies(policies.JSON200))
+	} else {
+		log.Printf("[WARN] autoscaling policies are not available for cluster %q: %v", data.Id(), checkErr)
+	}
+
 	return nil
 }
 
+func flattenAutoscalerPolicies(readPol *sdk.PoliciesConfig) []map[string]interface{} {
+	p := make(map[string]interface{})
+
+	p[PolicyFieldEnabled] = readPol.Enabled
+	p[PolicyFieldClusterLimits] = flattenAutoscalerPoliciesClusterLimits(readPol)
+	p[PolicyFieldNodeDownscaler] = flattenAutoscalerPoliciesDownscaler(readPol)
+	p[PolicyFieldSpotInstances] = flattenAutoscalerPoliciesSpot(readPol)
+	p[PolicyFieldUnschedulablePods] = flattenAutoscalerPoliciesUnsched(readPol)
+
+	return []map[string]interface{}{p}
+}
+
+func flattenAutoscalerPoliciesClusterLimits(readPol *sdk.PoliciesConfig) []map[string]interface{} {
+	p := make(map[string]interface{})
+
+	limits := make([]interface{}, 0, 1)
+	limits = append(limits, map[string]interface{}{
+		PolicyFieldClusterLimitsCPUmax: readPol.ClusterLimits.Cpu.MaxCores,
+		PolicyFieldClusterLimitsCPUmin: readPol.ClusterLimits.Cpu.MinCores,
+	})
+
+	p[PolicyFieldEnabled] = readPol.ClusterLimits.Enabled
+	p[PolicyFieldClusterLimitsCPU] = limits
+
+	return []map[string]interface{}{p}
```

Comment:
```suggestion
        return []map[string]interface{
           ...
           PolicyFieldClusterLimitsCPU: []interface{
              map[string]interface{}{
		PolicyFieldClusterLimitsCPUmax: readPol.ClusterLimits.Cpu.MaxCores,
		PolicyFieldClusterLimitsCPUmin: readPol.ClusterLimits.Cpu.MinCores,
             }
           }
        }
```

Also not sure if it's worth having a single-use remap in multiple methods, just an unnecessary boilerplate, just have one bigger `flattenAutoscalerPolicies`